version/1/3/0/jp/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+# 行動の貢献者コード
+
+貢献者とこのプロジェクトのメンテナ, としてとの利益のために
+オープンと歓迎のコミュニティを育成,我々はすべて尊重することを誓います
+レポートの問題を通じて貢献する人,ポスティング機能
+要求は,プル要求またはパッチを提出し,ドキュメントを更新
+やその他の活動.
+
+私たちはこのプロジェクトAに参加をすることにコミットしています
+関係なくレベルのすべての人のための嫌がらせのない経験
+経験, 性別、ジェンダーアイデンティティと表現,性的指向,
+障害者, 容姿, 体の大きさ, 人種, 民族, 年齢,宗教,国籍.
+
+参加者による容認できない行動の例としては、:
+
+* 性的言語や画像の使用
+* パーソナル 攻撃
+* 侮辱/軽蔑のコメント
+* パブリック または プライベート 嫌がらせ
+* このような物理的または電子などの他の個人情報を, 公開
+アドレス、許可なしに
+* その他の専門外の行為
+
+プロジェクトのメンテナは, 編集を削除する権利と責任を持って,
+やコメント、コミット,コード, ウィキの編集、問題を拒否し,他の
+この行動規範に合っていないの貢献は,または禁止します
+他の行動のための一時的または恒久的に任意の貢献者という彼ら
+不適切な,脅迫,攻撃,または有害を考えます.
+
+この行動規範を採用することにより、プロジェクトのメンテナは自分自身をコミット
+公正かつ一貫のあらゆる側面にこれらの原則を適用します
+このプロジェクトを管理します.従うか, または強制しないプロジェクトのメンテナ
+行動規範は, 恒久的にプロジェクトチームから除去することができます.
+
+この行動規範は、プロジェクトのスペース内や公共の両方で適用されます
+スペースは、個々のプロジェクトやそのコミュニティを代表したとき.
```

Comment:
As example, This meant to apply the conduct when people presenting name of project on their bio.